DataImport.Server.TransformLoad/Features/LoadResources/FileProcessor.cs

```diff
@@ -352,7 +352,8 @@ private Task<(RowResult, IngestionLogMarker)> MapAndPostCsvRow(IOdsApi odsApi, D
 
                 try
                 {
-                    mappedRow = TransformCsvRow(map, currentRow, rowNum, file);
+                    var endpointUrl = $"{odsApi.Config.ApiUrl}{map.ResourcePath}";
+                    mappedRow = TransformCsvRow(map, currentRow, rowNum, file, endpointUrl);
                 }
```

Comment:
@lchavarriaga 
We don't need any new type implementation and passing from File processor. We are catching the "Look up missing" error on line 357. There we are just logging, instead we need to create  something like : Task.FromResult((RowResult.Error, (IngestionLogMarker) null)) and return it. In calling place this will be logged automatically !!